tests/assembly/rust-abi-arg-attr.rs

```diff
@@ -0,0 +1,111 @@
+//@ assembly-output: emit-asm
+//@ revisions: riscv64 riscv64-zbb loongarch64
+//@ compile-flags: -C opt-level=3
+//@ [riscv64] only-riscv64
```

Comment:
With the cross-compilation flags set up properly, and now this being `no_core`, this no longer needs the "only" tags:
```suggestion
```

Reply:
Get it! Commits updated.